### DISCUSSION TITLE
[AnalyticalTable] How to modify the width of the selection column?

### DISCUSSION THREAD

**Discussion by ThiagoSzz** [UI5/webcomponents-react#6447] [Q&A] [answered]
Is it possible to modify the width of the selection column that is created by the component when the `selectionBehavior` prop is set to `Row`? Since it is a column generated by the component, it is not possible to change its width using the `width` prop like for custom columns.

**Comment by Lukas742** [comment, marked as answer]
Hi @ThiagoSzz, adjusting the column width of the selection column is not supported. You could change it with a [plugin hook](https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-analyticaltable-recipes--docs), but our calculation for the dynamic width of the columns doesn't react to static column width changes, so it will most likely not work correctly.
